logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt

```diff
@@ -301,6 +303,25 @@ internal class ConversationMapperImpl(
         ConversationOptions.Protocol.MLS -> ConvProtocol.MLS
     }
 
+    override fun toDaoModel(conversation: Conversation): ConversationEntity = with(conversation) {
```

Comment:
Shouldn't we pass during migration some more detailed object? I feel like using `Conversation` we miss some crucial data that we could and should get from the previous app, like `creatorId`, `accessRole` or `mutedStatus`, especially when we migrate it after the sync, so we should take all important data from the previous app, or am I misunderstanding this?

Reply:
I did the change and map the rest of the meaningful data we need, except for `accessRole`.
Talking with Marco, this case of migrated conversations that will not come from backend/sync, means are conversations without access anymore, so anyway we will not able to do nothing.

Nevertheless, I mapped the rest of the data to do it more generic